healthy/service_bridge.go

```diff
@@ -88,10 +88,7 @@ func (m *Monitor) fetchCheckForService(svc *service.Service, disco discovery.Dis
 	check := &Check{}
 	check.Type, check.Args = disco.HealthCheck(svc)
 	if check.Type == "" {
-		log.Warnf(
-			"Adding check for service %s (id: %s) failed. Got check type: %s, check args: %s.",
-			svc.Name, svc.ID, check.Type, check.Args,
-		)
+		log.Warnf("Got empty check type for service %s (id: %s)!", svc.Name, svc.ID)
```

Comment:
It looks to me like this got a little orphaned in the refactor at some point. But the new message should still show the args in case some were returned.

Reply:
Indeed, the message is a bit redundant, since we'll get [this one](https://github.com/Nitro/sidecar/blob/a6345e5f19c7ebb187fb48f6f06f6f070e88a410/healthy/service_bridge.go#L133) anyway.

I got rid of it completely.